core/config/schema.go

```diff
@@ -170,20 +172,23 @@ type ConfigSchema struct {
 	OCR2MonitoringEndpoint                 string        `env:"OCR2_MONITORING_ENDPOINT"`
 
 	// OCR V1
-	OCRBlockchainTimeout                  time.Duration `env:"OCR_BLOCKCHAIN_TIMEOUT" default:"20s"`
+	// Per-chain defaults
```

Comment:
Making it clear that the ones which don't have all-chain defaults here have per-chain defaults